security/pkg/server/ca/server.go

```diff
@@ -47,20 +47,22 @@ const (
 
 type authenticator interface {
 	Authenticate(ctx context.Context) (*authenticate.Caller, error)
+	AuthenticatorType() string
 }
 
 // Server implements IstioCAService and IstioCertificateService and provides the services on the
 // specified port.
 type Server struct {
+	monitoring     monitoringMetrics
```

Comment:
Why do you reorder the fields?

Reply:
There was some linter error and this would fix it. Basically re-order the struct would save some memory due to padding.